### PR TITLE
Add ui feature - viewD unfinished

### LINKED ISSUE #53: Delivery Tasks - View unfinished delivery tasks
As a Driver supervisor I can view unfinished delivery tasks so that I know which task needs to be worked on.

### PR DESCRIPTION
**Pull request by Krit-K**
Closes #53

### CODE REVIEW

File: src/main/java/seedu/address/ui/TaskListPanel.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.task.Task;
+
+/**
+ * Panel containing the list of tasks.
+ */
+public class TaskListPanel extends UiPart<Region> {
+    private static final String FXML = "TaskListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(TaskListPanel.class);
```
Comment (tysng) [inline src/main/java/seedu/address/ui/TaskListPanel.java]: Logger seems to be irrelevant? I'm not very familiar with the UI code, so disregard this if I'm wrong.

**Review by tysg — CHANGES_REQUESTED** [review]
Good job on getting the UI to work! Just a few things to address...

**Comment by junnbang** [comment]
Kris: view command unfinished

**Review by junnbang — DISMISSED** [review]
looks good

**Review by tysg — APPROVED** [review]

**Pull request merged**